internal/migrations/sql.go

```diff
@@ -28,24 +57,37 @@ var (
 
 	// getMigrationSQL is the sql query used to get the current migration.
 	getMigrationSQL = `SELECT start_height, end_height, chain_id FROM ` + migrationsSchemaName + `.migration;`
-	// migrationIsActiveSQL is the sql query used to check if a migration is active.
+	// migrationIsActiveSQL is the sql query used  to check if a migration is active.
```

Comment:
I believe the answer is yes, but just to confirm, this schema is part of statesync / snapshotting right?

Reply:
Yup, it is.